war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1231,12 +1222,6 @@ function rowvgStartEachRow(recursive,f) {
         form = null; // memory leak prevention
     });
 
-    // hook up tooltip.
-    //   add nodismiss="" if you'd like to display the tooltip forever as long as the mouse is on the element.
-    Behaviour.specify("[tooltip]", "-tooltip-", ++p, function(e) {
-        applyTooltip(e,e.getAttribute("tooltip"));
-    });
```

Comment:
Why migrate away from applying tooltips via `Behavior#specify` / `Behavior#applySubtree`?

Reply:
Rewritten it to use Behaviour 👍